templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -191,8 +201,9 @@ Resources:
     Properties:
       TemplateURL:
         !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/templates/quickstart-postgres-for-atlassian-services.yaml
-        - QSS3Region: !If ["GovCloudCondition", "s3-us-gov-west-1", "s3"]
+        - https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/templates/quickstart-postgres-for-atlassian-services.yaml
+        - S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
```

Comment:
Will this already work, i.e. is there already an `aws-quickstart-us-east-1` bucket?

Reply:
I dont think so, additionally these new URL's may change as all of this stuff is finalized on the AWS side

Reply:
Correct:
`https://aws-quickstart-us-east-1.s3.us-east-1.amazonaws.com/quickstart-atlassian-crowd/submodules/quickstart-atlassian-services/templates/quickstart-postgres-for-atlassian-services.yaml`